src/utilities/fetch-package-readmes.mjs

```diff
@@ -34,6 +34,9 @@ const loaderGroup = {
 const communityPackages = [{
   name: 'svg-chunk-webpack-plugin',
   contributors: ['yoriiis', 'alexander-akait']
+}, {
+  name: 'chunks-webpack-plugin',
+  contributors: ['yoriiis', 'alexander-akait']
```

Comment:
Do we need this now? We have directly linked to the github readme docs in `split-chunks-plugin.mdx`

Reply:
It seems interesting to me to keep it, it allows you to directly see who contributed to the project from the plugin's webpack doc.

This is also the case on `svg-chunk-webpack-plugin`.